pennylane/transforms/tape_expand.py

```diff
@@ -435,14 +435,15 @@ def new_preprocess(execution_config=qml.devices.DefaultExecutionConfig):
                 container.kwargs["max_expansion"] = decomp_depth
 
                 for cond in ["stopping_condition", "stopping_condition_shots"]:
-                    original_stopping_condition = container.kwargs[cond]
+                    if cond in container.kwargs:
```

Comment:
Under what condition are these not in the `container.kwargs`? Just wondering if it could fit in a comment here for future reference

Reply:
Added